README.md

```diff
@@ -1,6 +1,6 @@
-#powershell
+# PowerShell
```

Comment:
This is typically lowercase, but you will need to ask @jbondpdx why as a refresher. I believe it is to show this is the provider/module versus the underlying technology.